examples/common/pigweed/rpc_services/Device.h

```diff
@@ -481,6 +482,22 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         return pw::OkStatus();
     }
 
+    virtual pw::Status TriggerIcdCheckin(const pw_protobuf_Empty & request, pw_protobuf_Empty & response)
+    {
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+        chip::DeviceLayer::PlatformMgr().ScheduleWork(
+            [](intptr_t) {
+                ChipLogDetail(AppServer, "Being triggerred to send ICD check-in message to subscriber");
+                chip::app::ICDNotifier::GetInstance().NotifyNetworkActivityNotification();
```

Comment:
@mkardous-silabs @jmartinez-silabs  can this trigger the check-in no matter whether we have tracking subscription in icd device or not? I remember when I trigger button 0 in silab board brd4187c +brd4002a, I can see check-in message no matter what.